src/object/srv_ec_aggregate.c

```diff
@@ -2506,14 +2513,23 @@ ds_obj_ec_aggregate(void *arg)
 	struct ds_cont_child	*cont = arg;
 	struct ec_agg_param	agg_param = { 0 };
 	struct agg_param	param = { 0 };
+	int			rc;
 
 	D_DEBUG(DB_EPC, "start EC aggregation "DF_UUID"\n",
 		DP_UUID(cont->sc_uuid));
 	param.ap_data = &agg_param;
 	param.ap_start_eph_get = cont_ec_agg_start_eph_get;
 	param.ap_cont = cont;
 	param.ap_req = cont->sc_ec_agg_req;
-	ec_agg_param_init(cont, &param);
+	rc = ec_agg_param_init(cont, &param);
+	if (rc) {
+		/* To make sure the EC aggregation can be run on this xstream,
+		 * let's do not exit here, and in cont_ec_aggregate_cb(), it will
+		 * keep retrying parameter init.
+		 */
+		D_DEBUG(DB_EPC, DF_UUID" EC aggregation failed: %d\n",
```

Comment:
Should it be an error message or debug message?

Reply:
Should it use D_CDEBUG then?

e.g.
D_CDEBUG(rc == -DER_NOTLEADER || ..., DB_EPC, DLOG_ERR, "EC aggregation failed: "DF_RC"\n", DP_RC(rc));

Note the usage of DF_RC.  I think at the very least, we should do that.